lib/volcanic/authenticator/v1/scope.rb

```diff
@@ -44,10 +47,37 @@ def to_s
         base
       end
 
+      def <=>(other)
+        score < other.score
+      end
+
+      # For each element in the VRN, assign it a value
+      # element = * (0)
+      # element is value (0.5)
```

Comment:
0.5 no longer appears

...and the rest aren't multipliers.... may want renaming